src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends FigureColor {
```

Comment:
The class name 'FigureColor' seems to be a verb which is not recommended for class names. Class names should be nouns. If 'FigureColor' is an interface meant to define a behavior, consider renaming it to something like 'Colorable'.